spi/build.gradle

```diff
@@ -36,11 +39,83 @@ dependencies {
     compileOnly "org.elasticsearch:elasticsearch:${es_version}"
     compile "com.cronutils:cron-utils:7.0.5"
 
-    testImplementation "org.elasticsearch.test:framework:${es_version}"
+    testImplementation "org.elasticsearch.test:framework:7.1.0"
```

Comment:
Is there a reason we opted to use ```7.1.0``` for the test framework here?

Reply:
There were some issues with using the 6.8 testing framework in the sample plugin extension. After discussion with Yan this was a workaround for now since it doesn't seem like a blocker for job scheduler itself.